wp/resources/sample_binaries/goto_string/Makefile

```diff
@@ -7,7 +7,7 @@ $(BIN_DIR):
 	mkdir -p $@
 
 $(BIN_DIR)/%.s: $(SRC_DIR)/%.c $(BIN_DIR)
-	gcc -S -Wall -g -o $@ $<
+	gcc -S -Wall -g -w -o $@ $<
```

Comment:
Quick question about this change. Doesn't `-Wall` turn on all warnings and `-w` suppress warnings? Does removing `-Wall` not work?

Reply:
You're right! Silly me. I will delete the `-Wall` and the `-w`.